modules/system/classes/MediaLibraryItem.php

```diff
@@ -68,14 +73,16 @@ class MediaLibraryItem
 
     /**
      * @param string $path
+     * @param string $dimensions
      * @param int $size
      * @param int $lastModified
      * @param string $type
      * @param string $publicUrl
      */
-    public function __construct($path, $size, $lastModified, $type, $publicUrl)
+    public function __construct($path, $dimensions, $size, $lastModified, $type, $publicUrl)
```

Comment:
We should be passing this as the last argument for maximum BC, and since it's only relevant to specific types of media items we should start moving towards metadata support, so the parameter should actually be `array $metadata` and contain `'dimensions'` as a an array with `'width'` and `'height'` values.